app/javascript/components/App.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
+import SideNav from "./shared/side_nav";
+import Developer from "./developer/Developer";
+import MainPage from "./developer/MainPage";
+import AddBooking from "./booking/AddBooking";
+import Bookings from "./booking/Bookings";
+import AddDev from "./developer/AddDev";
+import DeleteDevelopers from "./developer/DeleteDevelopers";
+import Reserve from "./booking/Reserve";
+
+function App() {
```

Comment:
- [ ] Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latests standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can make a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.